Paco-Server/git-hooks/pre-commit

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+source ~/.bash_profile
```

Comment:
I do not have any specific statements in .bash_profile. This file is supposed to be saved in the .git/hooks folder. Since, we cannot source control hooks folder inside .git, I just thought we can add it here. From here, we can have either symlinks to make git look in this folder, or we can manually paste this file in .git/hooks folder. I took the latter approach.